devsetup/scripts/gen-edpm-compute-node.sh

```diff
@@ -157,17 +158,16 @@ cat <<EOF >${OUTPUT_BASEDIR}/${EDPM_COMPUTE_NAME}.xml
 EOF
 
 cat <<EOF >${OUTPUT_BASEDIR}/${EDPM_COMPUTE_NAME}-firstboot.sh
-growpart /dev/vda 1
-xfs_growfs /
+# extend GPT partition to the end of the disk, can be removed when this lands:
+# https://review.opendev.org/c/openstack/diskimage-builder/+/827558
+sgdisk -e /dev/vda
```

Comment:
the diskimage-builder change has landed, so there would be no need for any firstboot.sh with this image now